fortuna/src/login/Navbar.js

```diff
@@ -0,0 +1,33 @@
+//@flow strict
+
+import * as React from 'react';
+import '../Main.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+// Navbar component. Displays back button and player's name and currency.
+//
+// The Navbar has 3 columns:
+// The left column is the back button.
+// The middle column is the current page name.
+// The right column is the player's name and current currency.
+//
+// The columns must be customized for each page this component is used for.
+class Navbar extends React.Component<{||}> {
+	render(): React.Node {
+		return (
+			<div className="topnav">
+				<div className="navleft">
+					<button type="button" className="navbtn">&#60;&#45; Leave</button>

```

Comment:
This would be a good thing to have be passed in as part of the Props. Thanks for making its own component though